### DISCUSSION TITLE
SMTP issues with Docker Compose - no emails

### DISCUSSION THREAD

**Discussion by AGKavanagh** [Leantime/leantime#1280] [Q&A]
Hello! I've seen a few discussions and questions with similar issues but I can't seem to find a fix that works in my case.

I'm trying to get Leantime up and running via docker compose, and while everything else is running fine no emails are being sent out, which of course is stopping me from adding other users.

In the docker-compose.yml file I have the following variables set:
      LEAN_EMAIL_RETURN: 'leantime@mycompany*.ie'
      LEAN_EMAIL_USE_SMTP: true
      LEAN_EMAIL_SMTP_HOSTS: 'smtp.mycompany*.eu'
      LEAN_EMAIL_SMTP_PORT: 25

The SMTP host does not use authentication. Even when I add
      LEAN_EMAIL_SMTP_USERNAME: ''
      LEAN_EMAIL_SMTP_PASSWORD: ''
it still doesn't work. These details work with other docker containers on the same server so isn't not an issue with the host.

What am I doing wrong? Or is this a case of only SMTP hosts that require authentication can be used?

Thank you

**Comment by marcelfolaron** [comment]
Hi.
To turn off authentication please use
LEAN_EMAIL_SMTP_AUTH: false

**Reply by AGKavanagh** [reply]
I've tried that now and it works, thank you very much
